### ISSUE TITLE
Play Store sign-in stuck on "Checking info..."

### ISSUE THREAD

**Issue by RogueOwl** [waydroid/waydroid#741]
**Describe the bug**
Title says all, can't sign in to my Google account through either the Play Store or system settings

**General information (please complete the following information):**
 - Waydroid tools Version 1.3.4
 - Waydroid Images Version GAPPS 20230128

**Desktop (please complete the following information):**
 - OS: EndeavourOS
 - GPU: NVIDIA RTX 3080 Laptop GPU (or AMD Radeon integrated GPU)
 - Kernel version: Zen 6.1.9
 - Host mesa version: [e.g. mesa 2.2]
 - Desktop: KDE Plasma 5.26.5

**Additional context**
I've tried fixes listed on various sites that say to clear the app data and app preferences, but neither worked
[waydroid_base.prop.txt](https://github.com/waydroid/waydroid/files/10611270/waydroid_base.prop.txt)
[waydroid.log.txt](https://github.com/waydroid/waydroid/files/10611272/waydroid.log.txt)
[waydroid.logcat.txt](https://github.com/waydroid/waydroid/files/10611273/waydroid.logcat.txt)

**Comment by skoruppa** [comment]
Just updated Waydroid image and I have the same issue. After confirming the login, I'm stuck on a loading screen 
![obraz](https://user-images.githubusercontent.com/899429/217376350-d5ca2b03-ff34-4ae3-b9de-922615dcbd83.png)

**Comment by HrBingR** [comment]
Same issue here, unfortunately.

**Comment by deadghost** [comment]
Installed waydroid today and had the same issue. I already tried to query out the android_id according to https://docs.waydro.id/faq/google-play-certification and it's just not there.

```
sqlite3 gservices.db "select * from main LIMIT 10;"
digest|1-REDACTEDda39a3ee5e6bbfef95601890REDACTED
```

I'm hard stuck on setting up the play store as it is.

**Comment by flxjoeken** [comment]
I've got the same issue

**Comment by artofey** [comment]
Hi there!
Is there any solution? No android_id

**Comment by t0stiman** [comment]
Did you guys ever find a solution?

**Comment by HrBingR** [comment]
> Did you guys ever find a solution?

Yes, for me it ended up being a firewall problem. Disabled firewalld to test and it immediately worked.

**Comment by MegaShinySnivy** [comment]
Same issue here, no firewall, just a vanilla image with gapps.

**Comment by joser93** [comment]
Same issue and no device_id found, any help?

**Comment by jarommadsen** [comment]
Same issue

**Comment by Chubukov-Aleksey** [comment]
I also have experienced this issue. It appears to be a networking issue. After removing docker and deleting the `docker0` interface, Play Store started working properly. 
Kernel: 6.9.7-200.fc40.x86_64 arch: x86_64 bits: 64
Distro: Fedora Linux 40 (KDE Plasma)

**Comment by galemwendar** [comment]
@Chubukov-Aleksey, thanks, solved my problem

**Comment by awcator** [comment]
I was able to ping the Linux host machine, but unfortunately, I couldn't ping any other servers like 1.1.1.1 from ADB. Adding iptables rules and enabling IP forwarding on the host machine resolved the issue for me. Hope this helps someone 

replace wlan0 with your interface
```bash
sudo sysctl -w net.ipv4.ip_forward=1
sudo iptables -t nat -A POSTROUTING -s 192.168.240.0/24 -o wlan0 -j MASQUERADE
sudo iptables -A FORWARD -i waydroid0 -o wlan0 -j ACCEPT
sudo iptables -A FORWARD -i wlan0 -o waydroid0 -m state --state ESTABLISHED,RELATED -j ACCEPT
```

**Comment by EvanEdwards** [comment]
Possibly related:  I just filed #2192 -- turning off a VPN seems to be needed for the initialization of the Google security features.

**Comment by emmanueltab** [comment]
for my case it was a firewall issue. I disabled my firewall and it immediately worked  without having to reboot waydroid. I am using Linux mint xfce and running waydroid inside Weston. 

try turning off your firewall, then try to use the google apps again. If it works, then you can try these commands to allow waydroid with your firewall still on:

from google AI:
sudo ufw route allow in on waydroid0
sudo ufw allow in on waydroid0
sudo ufw reload

**Comment by utomsig** [comment]
> for my case it was a firewall issue. I disabled my firewall and it immediately worked without having to reboot waydroid. I am using Linux mint xfce and running waydroid inside Weston.
> 
> try turning off your firewall, then try to use the google apps again. If it works, then you can try these commands to allow waydroid with your firewall still on:
> 
> from google AI: sudo ufw route allow in on waydroid0 sudo ufw allow in on waydroid0 sudo ufw reload

I did run those commands, but after ufw reload I got the output
Firewall not enabled (skipping reload)
So I had to to
`sudo ufw enable`
to get to the Google play store. Though I first got a prompt inside Waydroid to update. After succeeding with that and a few attempts to restart the Play store I was in. 
A one liner of what I did is:
`sudo ufw route allow in on waydroid0 &&  sudo ufw allow in on waydroid0 &&  sudo ufw reload && sudo ifw enable`
